AuditRulesMonitor.cpp

```diff
@@ -0,0 +1,228 @@
+//
+// Created by tad on 3/18/19.
+//
+
+#include "AuditRulesMonitor.h"
+
+#include "Logger.h"
+#include "ExecUtil.h"
+
+void AuditRulesMonitor::run() {
+    Logger::Info("AuditRulesMonitor: Starting");
+    while(!_sleep(1000)) {
```

Comment:
Should these time constants be defined elsewhere?

Reply:
The values are only used in one location, so I'm ok with not making consts.